modules/rgbd/src/pose_graph.cpp

```diff
@@ -4,44 +4,75 @@
 
 #include "pose_graph.hpp"
 
+#include <fstream>
 #include <iostream>
 #include <limits>
 #include <unordered_set>
 #include <vector>
 
-#if defined(CERES_FOUND)
-#include <ceres/ceres.h>
+// a very stupid workaround against unreachable code warning
+#if defined(_MSC_VER)
+#pragma warning(disable : 4702)
```

Comment:
It makes sense to put `PoseGraph::optimize()` under `#ifdef HAVE_EIGEN` condition if Eigen is mandatory here.

BTW, Accurate condition should have `&& !defined(HAVE_EIGEN)` too.